src/character.cpp

```diff
@@ -2267,10 +2267,6 @@ bionic_id Character::get_remote_fueled_bionic() const
 
 bool Character::can_fuel_bionic_with( const item &it ) const
 {
-    if( ( !it.is_fuel() && !it.type->magazine && !it.flammable() ) || it.is_comestible() ) {
```

Comment:
Are you sure that removing the check for is_fuel() is not going to cause issues? I think the code for fuel consumption expects to find fuel data further down the line

Reply:
hm, i see. that could end up being bad if anyone does any hinky json...

Reply:
Would this change be the problem battery and ethanol CBMs are having now?

Reply:
which would be...?

Reply:
Battery CBM doesn't work: https://github.com/CleverRaven/Cataclysm-DDA/issues/49509